src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -448,11 +398,11 @@ const useGroupInfo = () => {
       }. Please type reason for reporting this group below. Thank you!`
     });
   };
-
   const handlePressContact = async (item) => {
-    if (item.user_id === profile.myProfile.user_id) return;
+    const isAnonymousUser = !isContainUrl(item?.user?.image) || item?.user?.name === ANONYMOUS_USER;
+    if (item?.user_id === profile?.myProfile?.user_id) return;
 
-    if (anonUserEmojiName) {
+    if (anonUserEmojiName || isAnonymousUser) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition for checking if a user is anonymous has been updated. The new condition checks if the user's image URL contains a URL or if the user's name is equal to `ANONYMOUS_USER`. Please ensure that this new condition correctly identifies anonymous users.

```diff
- if (anonUserEmojiName) {
+ const isAnonymousUser = !isContainUrl(item?.user?.image) || item?.user?.name === ANONYMOUS_USER;
+ if (anonUserEmojiName || isAnonymousUser) {
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->